### PR TITLE
make all fields as not null in lnk-user-organization

### PR DESCRIPTION
**Pull request by nulls**

### CODE REVIEW

**Review by sanyavertolet — APPROVED** [review]

**Review by kgevorkyan — APPROVED** [review]

File: db/v-2/tables/lnk-user-organization.xml [hunk 1]
```diff
@@ -58,4 +58,11 @@
       onDelete="CASCADE"/>
   </changeSet>
 
+  <changeSet id="lnk-user-organization-4" author="nulls">
+    <delete tableName="lnk_user_organization">
+      <where>role is NULL</where>
```
Comment (Cheshiriks) [inline db/v-2/tables/lnk-user-organization.xml:63]: I don't think it should be removed.
Can be replaced with NONE role

File: db/v-2/tables/lnk-user-organization.xml [hunk 2]
```diff
@@ -58,4 +58,11 @@
...
+    </delete>
+    <addNotNullConstraint tableName="lnk_user_organization" columnName="role"/>
```
[the remark this answers is not in the document]
Comment (Cheshiriks) [reply db/v-2/tables/lnk-user-organization.xml:65]: This is strange logic. We have a dictionary of roles, but some roles should not be written to the database. We will have to add such a check everywhere. And if we forget it and this role is still recorded in the database?

[the remark this answers is not in the document]
Comment (nulls) [reply db/v-2/tables/lnk-user-organization.xml]: I guess this role was added to resolve null checking issues. 
It's not a problem to write it by mistake (I think), but it's not needed to write it explicitly.

Nonetheless, we don't need to put this value as default to database. It should be set by BL. And this field should be required (it's a main column in this table)

[the remark this answers is not in the document]
Comment (Cheshiriks) [reply db/v-2/tables/lnk-user-organization.xml]: Ok. So be it.

**Comment by nulls** [comment]
Need to fix foreign key:
```
Caused by: java.sql.SQLException: Cannot drop column 'owner_id': needed in a foreign key constraint 'fk_organization_user'
```

**Review by akuleshov7 — APPROVED** [review]

**Pull request merged**